dll/win32/shell32/shlfileop.cpp

```diff
@@ -663,6 +663,40 @@ EXTERN_C DWORD WINAPI Win32DeleteFileW(LPCWSTR path)
     return (SHNotifyDeleteFileW(NULL, path) == ERROR_SUCCESS);
 }
 
+/************************************************************************
```

Comment:
```suggestion
#ifdef __REACTOS__
/************************************************************************
```

Reply:
Why ifdef reactos?
cpp files are not from wine?